mlos_bench/db/db_schema_sqlite.sql

```diff
@@ -0,0 +1,82 @@
+--
+-- Copyright (c) Microsoft Corporation.
+-- Licensed under the MIT License.
+--
+-- SQLite3/DuckDB version of the DB schema.
+
+DROP TABLE IF EXISTS trial_telemetry;
+DROP TABLE IF EXISTS trial_results;
+DROP TABLE IF EXISTS trial_config;
+DROP TABLE IF EXISTS experiment_merge;
+DROP TABLE IF EXISTS trial_status;
+DROP TABLE IF EXISTS experiment_config;
+
+CREATE TABLE experiment_config (
+    exp_id VARCHAR(128) NOT NULL,
+    descr TEXT,
+    git_repo VARCHAR(255) NOT NULL,
+    git_commit VARCHAR(40) NOT NULL,
+
+    PRIMARY KEY (exp_id)
+);
+
+CREATE TABLE trial_status (
+    exp_id VARCHAR(128) NOT NULL,
+    trial_id INTEGER NOT NULL,
```

Comment:
I think Sqlite also supports `GENERATED ALWAYS AS IDENTITY` syntax, but we'll have to check.
That seems slightly more preferable way to get an "auto-increment" column than just leaving it as "not-null" and expecting the driver to handle it for us.
Else, we should probably at least leave a comment in the schema that we expect that to be the case.